templates/tabletop-development.ini.erb

```diff
@@ -0,0 +1,6 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+[secret]
```

Comment:
I modelled this template off of what prime's uber instance had in its INI, so if nothing has changed between now and Feb, it should work however it was working before.